.golangci.yml

```diff
@@ -65,35 +65,40 @@ linters-settings:
     min-len: 3
     # minimal occurrences count to trigger, 3 by default
     min-occurrences: 3
+  misspell:
+    # Correct spellings using locale preferences for US or UK.
+    # Default is to use a neutral variety of English.
+    # Setting locale to US will correct the British spelling of 'colour' to 'color'.
+    locale: US
   lll:
     # max line length, lines longer will be reported. Default is 120.
     # '\t' is counted as 1 character by default, and can be changed with the tab-width option
     line-length: 120
     # allow long comments.
     exclude: '//'
-    # tab width in spaces. Default to 1.
-    tab-width: 4
-
 
 linters:
   enable-all: true
   disable:
-    - maligned
-    - prealloc
-    - lll
-    - gochecknoinits
+    - gosec
+    - bodyclose
```

Comment:
was that producing some error? in this lib we don't have a server.

Reply:
no. just copied it over from the bb app repo. removed.